phpBB/styles/prosilver/theme/common.css

```diff
@@ -1269,6 +1269,10 @@ ul.linklist:after,
 	padding: 4px 6px;
 }
 
+.badge.hidden {
+	display: none;
+}
+
```

Comment:
Right, that's why i said it's a separate issue... not for this PR. Personally I'd like to just use the existing hidden rule, but I think the styles team may not like using !important.